_inc/client/state/site/reducer.js

```diff
@@ -53,21 +53,21 @@ export const reducer = combineReducers( {
  * Returns true if currently requesting site data. Otherwise false.
  * otherwise.
  *
- * @param  {Object}  state Global state tree
- * @return {Boolean}       Whether site data is being requested
+ * @param  {Object}  siteData Current Data for this site.
+ * @return {Boolean}          Whether site data is being requested
  */
-export function isFetchingSiteData( state ) {
-	return !!state.jetpack.siteData.requests.isFetchingSiteData;
+export function isFetchingSiteData( siteData ) {
```

Comment:
Related to previous comments, the selector should be written as if it received the global `state` tree object